sbin/cubeit

```diff
@@ -890,15 +890,21 @@ case $TARGET_TYPE in
 	    fi
 	fi
 
-	if [ -n "${FDISK_PARTITION_LAYOUT_INPUT}" ];then
-	    debugmsg ${DEBUG_INFO} "$INSTALLERS_DIR/cubeit ${INSTALL_ROOTFS} \
-		    --config ${TARGET_CONFIG_FILE} $device --partition_layout ${FDISK_PARTITION_LAYOUT_INPUT}"
-	    $INSTALLERS_DIR/cubeit-installer -b --artifacts ${ARTIFACTS_DIR} \
-		    --config ${TARGET_CONFIG_FILE} --partition_layout ${FDISK_PARTITION_LAYOUT_INPUT} ${INSTALL_ROOTFS} $device
+	if [ -n "$INSTALLER_IMAGE" -a "$INSTALLER_IMAGE" == "t" ]; then
```

Comment:
This check doesn't make sense. It is either on, or off, So just test for nonzero.

Reply:
Yeah, you are right. I'll tune it and send V2 PR.
Thanks!

Fupan